src/functionalTest/resources/application.yaml

```diff
@@ -11,6 +11,9 @@ azure:
 
 test-url: ${TEST_URL:http://localhost:4013}
 
+case-flags:
```

Comment:
we don't need case flags changes here. remove it.

Reply:
Dont we need supplementary id for case flags?